packages/gatsby-plugin-i18n/package.json

```diff
@@ -26,7 +26,6 @@
     "@babel/register": "^7.12.1",
     "@babel/traverse": "^7.12.9",
     "@babel/types": "^7.12.7",
-    "intl-messageformat-parser": "^6.0.18",
```

Comment:
We are not using it?

Reply:
not directly, he is a dependency on react-intl